shell/platform/linux/fl_engine_private.h

```diff
@@ -175,6 +175,41 @@ GBytes* fl_engine_send_platform_message_finish(FlEngine* engine,
                                                GAsyncResult* result,
                                                GError** error);
 
+/**
+ * fl_engine_mark_texture_frame_available:
+ * @engine: an #FlEngine.
+ * @texture_id: an int64_t.
+ *
+ * Tells the Flutter engine that a new texture frame is available for the given
+ * texture.
+ *
+ * Returns: true on success.
+ */
+bool fl_engine_mark_texture_frame_available(FlEngine* engine,
```

Comment:
For consistency with the existing code this should probably return gboolean, and have the documentation "Returns: %TRUE on success".